### ISSUE TITLE
Test failure in OData connector

### ISSUE THREAD

**Issue by zregvart** [syndesisio/syndesis#4912]
## This is a...

<pre><code>
[ ] Feature request
[ ] Regression (a behavior that used to work and stopped working in a new release)
[x] Bug report  
[ ] Documentation issue or request
</code></pre>

## Description

Seems that we have a test failure, it's blocking the 1.6.1 release:

```
12:49:47 [ERROR] testReadMetaDataRetrievalWithSplit(io.syndesis.connector.odata.meta.ODataMetaDataRetrievalTest)  Time elapsed: 0.042 s  <<< ERROR!
12:49:47 java.lang.IllegalArgumentException: Class com.fasterxml.jackson.module.jsonSchema.types.ArraySchema not subtype of [simple type, class com.fasterxml.jackson.module.jsonSchema.types.ObjectSchema]
12:49:47 	at io.syndesis.connector.odata.meta.ODataMetaDataRetrievalTest.checkShape(ODataMetaDataRetrievalTest.java:68)
12:49:47 	at io.syndesis.connector.odata.meta.ODataMetaDataRetrievalTest.testReadMetaDataRetrievalWithSplit(ODataMetaDataRetrievalTest.java:176)
```

**Comment by heiko-braun** [comment]
ah, I triggered another build assuming it's a flaky test – apparently it's not

**Comment by zregvart** [comment]
I think it'll fail on this, it reliably fails for me locally.

**Comment by heiko-braun** [comment]
Ok, let’s wait until the fix is included


On 15. March 2019 at 13:43:10, Zoran Regvart (notifications@github.com)
wrote:

I think it'll fail on this, it reliably fails for me locally.

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub
<https://github.com/syndesisio/syndesis/issues/4912#issuecomment-473271267>,
or mute the thread

**Comment by zregvart** [comment]
Already fixed and backported via #4886 for #4833

**Comment by zregvart** [comment]
No need to verify this.
